### PR TITLE
Fixed tests, applied code style

### PR DESCRIPTION
**Pull request by jpatton-USGS**
This PR fixes the unit tests and applies the Google code format.

### CODE REVIEW

File: src/main/java/gov/usgs/locator/Stepper.java [hunk 1]
```diff
@@ -4,7 +4,6 @@
 import gov.usgs.locaux.Cratons;
 import gov.usgs.locaux.LocUtil;
 import gov.usgs.locaux.Slabs;
-// import gov.usgs.locaux.ZoneStats;
```
Comment (mguy-usgs) [inline src/main/java/gov/usgs/locator/Stepper.java:7]: Isn't Ray going to put Zone Stats back in...maybe you don't want to delete these zone stats things that are commented out...?

**Review by mguy-usgs — CHANGES_REQUESTED** [review]
This looked good, and I think you left the commented out zone stats stuff mostly in (the plus and minus red and green is sometimes confusing as to the end result of the actual file) but I *think* Ray was going to add it back in, so we don't want to remove the commented out zone stat stuff if you want Ray to start with this version. So I won't merge until you confirm....

File: src/main/java/gov/usgs/locator/Stepper.java (hunk 1 above)
Comment (jpatton-USGS) [reply src/main/java/gov/usgs/locator/Stepper.java:7]: The code style checker (required for successful build) does not like commented out imports, it automatically nuked them when I applied the style.

Comment (jpatton-USGS) [reply src/main/java/gov/usgs/locator/Stepper.java:7]: anyone running gradle builds will not get a jar until the style checker is happy, so I don't see a way around it.

**Pull request merged**